src/utils/hooks/useLoadApplication.tsx

```diff
@@ -30,6 +30,7 @@ const useLoadApplication = (props: useLoadApplicationProps) => {
       serial: serialNumber,
     },
     skip: triggerProcessing,
+    fetchPolicy: 'cache-and-network',
```

Comment:
Some like this (but not exactly this) added to the updateApplicationMutation: 
`refetchQueries: (mutationResult) [{ query: 'getApplication', variables: {serial: mutationResult.serial}}]`

Reply:
Okay, I've added an additional check to the `skip` parameter, so it won't re-run the query if the application has previously been loaded. So therefore it won't make server requests when moving between pages. It'll only re-load from the server when refreshing the page and (re-)loading an application -- which it should do, right?

We always want the latest version of the application when going to it, as there are many ways that it might have changed on the server, right?